src/main/java/org/traccar/protocol/Gt06ProtocolDecoder.java

```diff
@@ -217,7 +217,7 @@ private static boolean hasLbs(int type) {
         }
     }
 
-    private static boolean hasStatus(int type) {
+    private static boolean hasStatus(int type, boolean modelNt20) {
```

Comment:
No, I think we should pass the model.

Reply:
Yes, I thought of that, but then I would hardcode NT20 twice. Maybe I can create a function for the modelNt20.

Reply:
I think hardcoding in two places is ok. If it goes beyond, we can have a constant or something.